packages/web-service/src/pages/contact/additional-contacts/additional-applicant/add-additional-applicant.njk

```diff
@@ -1,5 +1,5 @@
 {% extends "yes-no.njk" %}
-{% set title = 'Do you want to add an alternative contact?' %}
+{% set title = 'Do you want to add an alternate applicant?' %}
 {% set radioHint = 'This alternative contact must be someone who is authorised to deal with this application on behalf of the intended licence-holder.' %}
```

Comment:
AC3 of the ticket _should_ state that this hint text should be changed to `This person will not hold the licence, but will receive messages about this application.` as per the prototype and prototype screenshot. It doesn't, but can we make the change here anyway. I'll update the ticket accordingly and throw a comment on it to explain.

Reply:
Will make the change